docs/core/stories.rst

```diff
@@ -28,8 +28,43 @@ This is a guide to the story data format.
 Format
 ------
 
+With Rasa Open Source 2.0 release we've introduced a YAML format as unified and the most extendable way to manage the training data.
```

Comment:
Let's try to target novice users more, maybe something like:

```
Format
-------
The training data uses the YAML format. You can split the training data among any number of YAML files (just make sure to mention all of them when you run the training).  

.. note ::

    The new training data format was introduced with Rasa Open Source 2.0. If you want to update from an older 
    training data format, you can find more information about the upgrade in our migration guide.


```
let's also:
- remove the note that is right before the `Format` and incorporate the info here
- I'd vote for not calling this `the Rasa Core dialogue system` (paragraph before format)

Reply:
+1 would call it `dialogue manager` or `dialogue management` or similar